### DISCUSSION TITLE
Is there any way to hide the separator for transparent windows?

### DISCUSSION THREAD

**Discussion by vishalbalaji** [nvim-tree/nvim-tree.lua#1702] [Q&A]
I use this highlight to hide the nvim-tree separator.

```lua
vim.api.nvim_set_hl(0, "NvimTreeWinSeparator", { fg = "bg" })
```

This works fine when the background is opaque, but does not update for transparent backgrounds. Setting `fg = "NONE"` only makes the separator revert back to its default color. Is there any way to hide it in transparent windows or just an option to disable it?

**Comment by alex-courtis** [comment]
Hint: you can use hitest.vim to show the actual values and a demo for all highlight groups `:help nvim-tree-highlight`

You can completely clear highlight groups and their links e.g. `:hi clear NvimTreeWinSeparator`

There are some nvim deprecation complications around the vertsplit / winseparator highlight groups, see: #https://github.com/nvim-tree/nvim-tree.lua/issues/1221 . Result is two groups for nvim-tree and two groups for nvim, with default links between them.

Experimentation, rerunning hitest to see the results:
* clear `NvimTreeVertSplit`
* clear `NvimTreeWinSeparator`
* clear `WinSeparator`
* clear `VertSplit`
